CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ if (NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
 endif()
 message(STATUS "Build configuration: " ${CMAKE_BUILD_TYPE})
 
-find_library(precice precice PATHS $ENV{PRECICE_ROOT}/build)
+find_library(precice precice PATHS $ENV{PRECICE_ROOT}/build $ENV{PRECICE_ROOT}/build/latest)
```

Comment:
If this is for Scons, the latest build link is usually `last`, not `latest`.

Reply:
Thank you.